protocol/protocol.proto

```diff
@@ -240,6 +240,8 @@ message SpacePayload {
         bytes channel_id = 2;
         EventRef origin_event = 3;
         int64 updated_at_event_num = 6;
+        bool autojoin = 7;
+        bool show_user_join_leave_events = 8;
```

Comment:
ChannelProperties channel_properties = 8;

Reply:
Then we can add ChannelProperties to the ChannelUpdate and the ChannelPayload.Inception and set them all on channel creation.